src/services/routes/org.routes.ts

```diff
@@ -1,16 +1,28 @@
 import type { RouteOptions } from 'fastify';
-import isValidDomain from 'is-valid-domain';
 
 import { HTTP_STATUS_CODES } from '../http.js';
 import type { PluginDone } from '../types/PluginDone.js';
 import { ORG_SCHEMA } from '../schema/org.schema.js';
 import type { FastifyTypedInstance } from '../fastify.js';
-
-export enum ProblemType {
-  MALFORMED_ORG_NAME = 'https://veraid.net/problems/malformed-org-name',
-  MALFORMED_AWALA_ENDPOINT = 'https://veraid.net/problems/malformed-awala-endpoint',
+import { createOrg } from '../../org.js';
+import { CreationProblemType } from '../../CreationProblemType.js';
+
+const ORG_ROUTES_ERROR_MAPPING: {
```

Comment:
Since this isn't exported, I'd suggest something along the lines of `RESPONSE_CODE_BY_PROBLEM_TYPE` or `RESPONSE_CODE_BY_PROBLEM`.